src/promptflow-evals/promptflow/evals/evaluate/_eval_run.py

```diff
@@ -285,42 +323,74 @@ def log_artifact(self, artifact_folder: str) -> None:
         if not os.listdir(artifact_folder):
             LOGGER.error("The path to the artifact is empty.")
             return
+        if not os.path.isfile(os.path.join(artifact_folder, EvalRun.EVALUATION_ARTIFACT)):
+            LOGGER.error("The run results file was not found, skipping artifacts upload.")
+            return
         # First we will list the files and the appropriate remote paths for them.
-        upload_path = os.path.basename(os.path.normpath(artifact_folder))
-        remote_paths = {"paths": []}
+        root_upload_path = posixpath.join("promptflow", 'PromptFlowArtifacts', self.info.run_name)

```

Comment:
Why do we need to upload all of pf artifacts ? Should it not be limited to just uploading instance_results.jsonl ?